cluster/config-defaults.yaml

```diff
@@ -172,6 +172,8 @@ skipper_redis_dial_timeout: "25ms"
 skipper_redis_pool_timeout: "250ms"
 skipper_redis_read_timeout: "25ms"
 skipper_redis_write_timeout: "25ms"
+skipper_redis_min_conns: 25
```

Comment:
Default min [is 100 ](https://github.com/zalando/skipper/blob/bd87e31f76fbbf400aaeb4f7a345e277df8f0028/net/redisclient.go#L111-L112)